src/platform/testing/e2e/auth.js

```diff
@@ -90,6 +90,7 @@ function logIn(token, client, url, level) {
     .url(`${E2eHelpers.baseUrl}${url}`)
     .waitForElementVisible('body', Timeouts.normal);
 
+  E2eHelpers.disableAnnouncements(client);
   setUserToken(token, client);
```

Comment:
This has to be here so that the change to local storage is applied to the right domain